33141: Add entry to intro about the removed collectd IPMI plugin. (#647)
freenas/freenas-docs: userguide/intro.rst

  :menuselection:`Reporting`
  now shows the compressed physical L2ARC size.

> * :file:`usr/local/lib/collectd/ipmi.so` and the **openipmi** package

Reviewer: ```suggestion
* The :literal:`openipmi` package and :file:`usr/local/lib/collectd/ipmi.so`
```
(Markup should be like a command, but it's not a command, so I used literal.)

>   were removed to disable the broken collectd IPMI plugin.

Reviewer: ```suggestion
  have been removed to disable the non-functional collectd IPMI plugin.
```
